docs/Advanced-Usage/Manager/Manager-Environment-Variables.rst

```diff
@@ -6,7 +6,7 @@ Manager Environment Variables
 This page contains a centralized reference for the environment variables used
 by the manager.
 
-.. _config-runtime:
```

Comment:
Remove this? It appears to be unused now the duplicate label is gone.

Reply:
There is another reference in `Advanced-Usage/Manager/Manager-Configuration-Files.rst` which results a duplicate label.

Reply:
This was more of a nit, but what i was saying was that the _new_ label is unused.

Reply:
There is no section that refers to `runfarm-prefix` right?

Reply:
Doesn't look like it.